docs/User/Troubleshooting.md

```diff
@@ -26,41 +38,39 @@ Error code: 6002, as shown in the following screenshot occurs when Swarm Learnin
 3.  Verify if the Swarm licenses are installed using APLS web management console. For more information, see APLS User Guide.
 
 
-## Installation of HPE Swarm Learning on air-gaped systems or if the Web UI Installer runs into any issue and not able to install
+## 2. Installation of HPE Swarm Learning on air-gaped systems or if the Web UI Installer runs into any issue and not able to install
 
 - Download the following from HPE My Support Center(MSC) on a host system that has internet access - tar file (HPE_SWARM_LEARNING_DOCS_EXAMPLES_SCRIPTS_Q2V41-11033.tar.gz) and the signature file for the above tar file.
 - Untar the tar file under `/opt/hpe/swarm-learning`.
 - Do a docker login from your host:
    `docker login hub.myenterpriselicense.hpe.com –u <YOUR-HPE-PASSPORT-EMAIL> -p hpe_eval`
 - Pull the signed Swarm Learning images from HPEs Docker Trust Registry (DTR):
    ```
-   docker pull hub.myenterpriselicense.hpe.com/hpe_eval/swarm-learning/sn:<latest Swarm Learning Version (for example, 1.2.0)>
+   docker pull hub.myenterpriselicense.hpe.com/hpe_eval/swarm-learning/sn:<latest Swarm Learning Version>
    docker pull hub.myenterpriselicense.hpe.com/hpe_eval/swarm-learning/sl:<latest Swarm Learning Version>
    docker pull hub.myenterpriselicense.hpe.com/hpe_eval/swarm-learning/swci:<latest Swarm Learning Version>
    docker pull hub.myenterpriselicense.hpe.com/hpe/swarm-learning/swop:<latest Swarm Learning Version>
+   
+   For eg: docker pull hub.myenterpriselicense.hpe.com/hpe_eval/swarm-learning/sn:1.2.0
    ```
 - Copy the tar file and Docker images to the air-gaped systems.
 
-## System resource issues if too many SLs are mapped to the same SN
+## 3. System resource issues if too many SLs are mapped to the same SN
 
 When configuring Swarm Learning you may encounter system resource issues if too many SLs are mapped to same SN. For example:
     ```
     “swarm.blCnt : WARNING: SLBlackBoardObj : errCheckinNotAllowed:CHECKIN NOT ALLOWED”
     ```
 The suggested workaround is to start with mapping 4 SL to 1 SN. Then after, slowly scale no of SLs to SN
 
-## SWCI waits for task-runner indefinitely even after task completed or failed
+## 4. SWCI waits for task-runner indefinitely even after task completed or failed
 
 User to ensure no failure in ML code before Swarm training starts. Check using `SWARM_LOOPBACK ENV` and ensure, user coderuns fine and local training completes successfully.
 
-# Generic troubleshooting tips
+## 5. Error while docker pull Swarm Learning images: 'could not rotate trust to a new trusted root'
```

Comment:
New issue. from latest user guide (troubleshooting) to be added.

"Unable to contact API-Server"